libs/blocks/video/video.js

```diff
@@ -1,10 +1,13 @@
-import { createIntersectionObserver, getConfig } from '../../utils/utils.js';
-import { applyHoverPlay, getVideoAttrs, applyInViewPortPlay } from '../../utils/decorate.js';
+import { getConfig } from '../../utils/utils.js';
+import { decorateAnchorVideo } from '../../utils/decorate.js';
 
-const ROOT_MARGIN = 1000;
-
-const loadVideo = (a) => {
-  const { pathname, hash, dataset } = a;
+export default function init(a) {
+  a.classList.add('hide-video');
+  if (!a.parentNode) {
+    a.remove();
+    return;
+  }
```

Comment:
I see that the codecov check is failing because these lines are not covered. Is it possible to add some unit tests?

Reply:
It's actually not, or I wouldn't know how. When attempting to write a test, I noticed it might not even be possible to get there to begin with? 🤔 
https://github.com/adobecom/milo/pull/2849#discussion_r1760982302

If you remove the parentElement of a node, doesn't the node within that element vanish as well? Or is it detached?

`a.parentNode.remove()` didn't do anything 🤷 So in the interest of time I didn't try further

Reply:
Not looking into the matter for 2-3 hours helps 😁 You just pass in a newly created tag that's not part of the document ... 🙄